server/utils/referralUtils.test.ts

```diff
@@ -247,7 +247,7 @@ describe('ReferralUtils', () => {
   })
 
   describe('viewReferralNavigationItems', () => {
-    it('returns navigation items for the view referral pages and sets the requested page as active', () => {
+    it('returns navigation items for the view referral pages with the refer paths and sets the requested page as active', () => {
       const mockReferralId = 'mock-referral-id'
```

Comment:
Might be worth nesting both conditions in a `describe` block because Refer isn't really a default journey (although I think it's okay to use it as the default in the controller tests!)

The `const mockReferralId` can move into the outer scope here